lib/active_shipping/carriers/ups.rb

```diff
@@ -511,6 +511,8 @@ def build_shipment_request(origin, destination, packages, options={})
           label_format = options[:label_format] ? options[:label_format].upcase : 'GIF'
           label_size = options[:label_size] ? options[:label_size] : [4, 6]
 
+          character_set = options[:character_set] ? options[:character_set] : 'eng'
```

Comment:
Not requesting the change, but a cool Ruby thing: `options.fetch(:character_set, 'eng')` will grab the value from the hash or return the 2nd arg as a default.

Reply:
👍

Reply:
And if you don't specify a 2nd arg to `#fetch`, it will raise an error rather than return nil if the key cannot be found.